### ISSUE TITLE
[🚀 Feature]: Use latest 18.x version

### ISSUE THREAD

**Issue by apupier** [SeleniumHQ/selenium#13344]
### Feature and motivation

follow-up https://github.com/SeleniumHQ/selenium/issues/12723

I think it will be nice to have it working with latest 18.x as currently blocked on 18.17.0 https://github.com/SeleniumHQ/selenium/blob/10561bc593771e415d837441660805388554e96c/WORKSPACE#L235 which is the latest version supported by bazelbuild/rules_nodejs 5.8.4 which is the latest 5.x version.

### Usage example

It will allow to test with the LTS version which receives security updates.

**Comment by apupier** [comment]
bazelbuild/rules_nodejs 6 has a reduced scope https://github.com/bazelbuild/rules_nodejs/tree/main#60-scope-reduction

aspect-build/rules_js is claiming to be an alternative https://github.com/aspect-build/rules_js#relationship-to-rules_nodejs

**Comment by apupier** [comment]
attempt to migrate: https://github.com/apupier/selenium/tree/13344-useLatest18 but currently failing with:

```
(base) apupier@localhost:~/git/selenium$ bazel build javascript/...
ERROR: Skipping 'javascript/...': error loading package under directory 'javascript': error loading package 'javascript/grid-ui': Unable to find package for @npm//react-scripts:index.bzl: The repository '@npm' could not be resolved: Repository '@npm' is not defined.
WARNING: Target pattern parsing failed.
ERROR: error loading package under directory 'javascript': error loading package 'javascript/grid-ui': Unable to find package for @npm//react-scripts:index.bzl: The repository '@npm' could not be resolved: Repository '@npm' is not defined.
INFO: Elapsed time: 0.094s
INFO: 0 processes.
FAILED: Build did NOT complete successfully (0 packages loaded)
    currently loading: javascript/chrome-driver ... (7 packages)
```

**Comment by titusfortner** [comment]
Yes, migrating to rules_js is on our list of things to do:
https://github.com/orgs/SeleniumHQ/projects/2?pane=issue&itemId=43566772

**Comment by titusfortner** [comment]
If you can help assist/collaborate on the move, come find us in Selenium Chat and we can discuss: https://www.selenium.dev/support/#ChatRoom

**Comment by harsha509** [comment]
Hi @apupier,

We’ve now migrated to rules_js. 

You can check the Node.js version being used in this [node.toolchain configuration](https://github.com/SeleniumHQ/selenium/blob/204fefd8aa5ece4dfae2ec0e6fae03946aa05038/MODULE.bazel#L42)

Closing as resolved!
